src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -811,7 +811,7 @@ private async Task UpdatePullRequestAsync(InProgressPullRequest pr, List<UpdateA
             TargetRepoDependencyUpdate targetRepositoryUpdates =
                 await GetRequiredUpdates(updates, _remoteFactory, targetRepository, targetBranch);
 
-            if (targetRepositoryUpdates.CoherencyCheckSuccessful && targetRepositoryUpdates.RequiredUpdates.Count < 1)
```

Comment:
Why was the coherency check here before?

Reply:
It was added recently in https://github.com/dotnet/arcade-services/commit/52bf5824a3b50bdcd996a5a9cffb8ce1accf14f1 and possibly started the bug